README.md

```diff
@@ -111,16 +118,26 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 ad5221e8ae73        ubuntu              "/bin/bash"         36 minutes ago      Up 36 minutes                     		suspicious_mirzakhani
 ```
 
-If this is your directory structure:
+Make sure that your `$SNAP_PATH` is set, if not:
 ```
-$GOPATH/src/github.com/intelsdi-x/snap/
-$GOPATH/src/github.com/intelsdi-x/snap-plugin-collector-docker/
+$ export SNAP_PATH=<snapDirectoryPath>/build
```

Comment:
Maybe this is an example?
https://github.com/intelsdi-x/snap-plugin-collector-influxdb